docs/rancher/csi-driver.md

```diff
@@ -143,6 +143,34 @@ The only difference is in generating the `cloud-init` config where you need to s
 ./generate_addon_csi.sh <serviceaccount name> <namespace> k3s
 ```
 
+## Customize the default StorageClass
+
+Harvester CSI Driver provides the interface to pre-define the default StorageClass. If we do not specify the default StorageClass, the default StorageClass of Harvester CSI Driver will be the same as Harvester cluster.
+
+To customize the default StorageClass, you can use the parameter `host-storage-class`. Please check with the following example:
+
+1. Create new a StorageClass of the host Harvester cluster
+
+  ![](/img/v1.3/rancher/create-new-sc.png)
+
+2. Deploy CSI driver with parameter `host-storage-class`
+
+  ![](/img/v1.3/rancher/deploy-csi-driver-with-host-storage-class.png)
+
+Then, make sure the Harvester CSI driver is ready. You can verify with the following steps.
+
+1. Create volume with harvester CSI driver
+
+  ![](/img/v1.3/rancher/create-volume-with-harvester-csi-driver.png)
```

Comment:
```suggestion
    1. On the **PersistentVolumeClaims** screen, create a PVC. Select **Use a Storage Class to provision a new Persistent Volume** and specify the StorageClass you created.
        
        Example:
        ![](/img/v1.3/rancher/create-volume-with-harvester-csi-driver.png)
```

Reply:
I changed the description to match what we expect users to do on the UI. In this case, they are supposed to create a PVC, not a volume. Also, there is no mention of the CSI driver on this screen.